aw_transform/chunk_events_by_key.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from typing import List, Dict
+
+from aw_core.models import Event
+
+logger = logging.getLogger(__name__)
+
+
+def chunk_events_by_key(events, key) -> List[Event]:
+    chunked_events = [] # type: List[Event]
+    for event in events:
+        if key not in event.data:
+            pass
+        elif len(chunked_events) == 0 or chunked_events[-1].data[key] != event.data[key]:
```

Comment:
One thing that always confused me about chunking before, and still does, is how it results in the timeline not showing any eventual gap between two events with the same chunk-key. Could we do anything about that? 

Maybe add a pulsetime-like argument that doesn't chunk events if separated by a significant time?

Reply:
Good point, will fix